### PR TITLE
Update readme with run config

### PR DESCRIPTION
**Pull request by BadrTad**
This PR updates the README.md, to document few changes about running the server with a configuration file,

### CODE REVIEW

File: be2-scala/README.md [hunk 1]
```diff
@@ -6,9 +6,29 @@ Proof-of-personhood, spring 2021: Scala language back-end
 ## Running the project
 
 There are two main possible ways of running the project :
-* import the project using IntelliJ
-* using **sbt**, execute `sbt compile` and `sbt run`
-
+* #### Option 1: Intellij
+> import the project using IntelliJ
```
[the remark this answers is not in the document]
Comment (BadrTad) [reply be2-scala/README.md:10]: Capture

[the remark this answers is not in the document]
Comment (BadrTad) [reply be2-scala/README.md:10]: This does not work when once applied and run.

[the remark this answers is not in the document]
Comment (RaulinN) [reply be2-scala/README.md:10]: Putting it right there as well for future referencement if needed

Screenshot 2021-12-23 at 17 45 09

File: be2-scala/README.md [hunk 2]
```diff
@@ -4,11 +4,39 @@ Proof-of-personhood, spring 2021: Scala language back-end
...
 
+---
 There are two main possible ways of running the project :
-* import the project using IntelliJ
-* using **sbt**, execute `sbt compile` and `sbt run`
-
+* #### Option 1: Intellij
+>1. Import the project using IntelliJ
+>2. Modify the default  Run configuration 'Server', to include the following __VM option__: <br>
+>__```-Dscala.config=src/main/scala/ch/epfl/pop/config```__
```
Comment (pierluca) [inline be2-scala/README.md:16]: why is it all quoted, i.e. prefixed with `>` ?

**Review by tuomas-p — COMMENTED** [review]
Not sure about the quotes (with >), like Pierluca already said but otherwise looks good! Useful explanation :slightly_smiling_face:

File: be2-scala/README.md (hunk 2 above)
Comment (BadrTad) [reply be2-scala/README.md:16]: For pretty formatting

**Review by pierluca — APPROVED** [review]

**Pull request merged**